tests/test-cli.js

```diff
@@ -0,0 +1,760 @@
+/* eslint-disable wrap-regex */
+"use strict"
+
+const loadFromDeps = (typeof process !== "undefined" && process.argv.length >= 2 && process.argv[1].match(/ospec\/node_modules\/\.bin\/ospec$/))
```

Comment:
Have you tested this on Windows outside of Cygwin/WSL/etc.?

Reply:
Of course not :-p

Good catch. This was added so that I can do both `npm run test` (which uses the ospec version in `node_modules`) and `node ./bin/ospec tests/test-*.js`, but I didn't think far enough, obviously.